README.md

```diff
@@ -7,7 +7,8 @@ as additional files will be in place. .rsyncignore is a pseudo
 .ignore file for rsync that lists files in this repo that should
 not be put onto downloads.keyman.com.
 
-For now, we only do manual uploads of files that change here.
+The _tools folder contains a script that determines which files should
```

Comment:
Are you planning to use syncfiles.txt as the rsync parameter?
```bash
--include-from=FILE
    This specifies a list of include patterns from a file.
```
imho, I wouldn't call it a "script".

Reply:
`_tools/syncfiles.txt` is a data file referenced in the script called `_tools/sync.ps1`.